### PR TITLE
Send certificate to node after dissemination + tests

### PR DESCRIPTION
**Pull request by zeegomo**

### CODE REVIEW

File: nomos-cli/Cargo.toml [hunk 1]
```diff
@@ -22,4 +22,5 @@ nomos-network = { path = "../nomos-services/network", features = ["libp2p"] }
 nomos-da = { path = "../nomos-services/data-availability", features = ["libp2p"] }
 nomos-libp2p = { path = "../nomos-libp2p"}
 nomos-core = { path = "../nomos-core" }
-full-replication = { path = "../nomos-da/full-replication" }
\ No newline at end of file
+full-replication = { path = "../nomos-da/full-replication" }
```
Comment (danielSanchezQ) [inline nomos-cli/Cargo.toml:25]: `git diff` drunkness

File: nomos-cli/src/cmds/disseminate/mod.rs [hunk 2]
```diff
@@ -185,7 +192,25 @@ impl ServiceCore for DisseminateService {
...
+                            if !res.status().is_success() {
+                                tracing::error!(
+                                    "Failed to send certificate to node: {}",
+                                    res.status()
+                                );
+                                std::process::exit(1);
+                            }
+                            terminal_cmd_done();
+                        }
+                    }
```
Comment (danielSanchezQ) [inline nomos-cli/src/cmds/disseminate/mod.rs:213]: If we do not send the certificate, shouldn't we at least save it or something? 🤔

Comment (zeegomo) [reply nomos-cli/src/cmds/disseminate/mod.rs:213]: right, probably

Comment (danielSanchezQ) [reply nomos-cli/src/cmds/disseminate/mod.rs:223]: Nitpick:  what if both options are activated? 🤔

**Review by danielSanchezQ — APPROVED** [review]
Looking good. Left a comment on usability but it can be checked later if needed.

**Pull request closed without merging**